MovingBlocks/Terasology
# Improve the biomes view

### Contains

Small improving of rendering biomes on GameDetailsScreen (biomes tab).
See also: https://github.com/Terasology/PolyWorld/pull/16

### How to test

1. Create/Load game with active PolyWorld module.
2. Open GameDetailsScreen and tab biomes.
3. Check that all biomes have the same style (no biomes in UpperCase).
